jobs/Llama2/finetune/eval_7b_hf.py

```diff
@@ -46,20 +48,19 @@ def generate_text_with_prompt(prompt, model, tokenizer, temperature=0.7):
 # Test with exactly same instructions used in fine-tuning
 instruction = "### Instruction: \nConvert the input English sentence to a Bliss sentence.\n\n"
```

Comment:
The `instruction` variable is not needed anymore, since the `instructions_map` is used instead, on the next and subsequent lines.

Reply:
Great catch, @klown. Now addressed. Thanks.